tracing-subscriber/src/fmt/mod.rs

```diff
@@ -297,6 +297,8 @@
 //!     https://docs.rs/tracing/latest/tracing/trait.Subscriber.html
 //! [`tracing`]: https://crates.io/crates/tracing
 //! [`fmt::format`]: mod@crate::fmt::format
+#[cfg(not(feature = "env-filter"))]
+use core::str::FromStr;
```

Comment:
nit, take it or leave it: we don't _need_ to import this from `core` here, since this whole module requires the `std` feature. importing it from `core`. this is fine either way, though.